tools/SetupFlow/DevHome.SetupFlow/Views/Summary/SummaryAppInstallationNotes.xaml.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using DevHome.SetupFlow.ViewModels;
+using DevHome.SetupFlow.Windows;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace DevHome.SetupFlow.Views.Summary;
+
+public sealed partial class SummaryAppInstallationNotes : UserControl
+{
+    public SummaryAppInstallationNotes()
+    {
+        this.InitializeComponent();
+    }
+
+    private void ViewAllButton_Click(object sender, Microsoft.UI.Xaml.RoutedEventArgs e)
+    {
+        // When installation notes' 'view all' button is clicked, open a new window with the full text
```

Comment:
Weird nit, but Visual Studio's spell checker tends to think apostrophes are part of the words. Are you getting spelling errors on "`'view`" and "`all'`"? I suggest just using quotation marks.

Reply:
I was not getting errors.  But I did make the suggested change.